### PR TITLE
belindas-closet-nextjs_6_321_mui-admin-page

### LINKED ISSUE #321: Admin Page Styling: Integrate MUI on the admin page
Relates to #320 
- [ ] Integrate MUI on the admin page.

### PR DESCRIPTION
**Pull request by nwm516**
Resolves #321 

Integrate MUI on the Admin page.

https://github.com/SeattleColleges/belindas-closet-nextjs/assets/36825393/80bd082a-8d3d-4ecc-a5bc-bfeed0810e78

Before: 
Admin Page  Placeholder Creator Page

After:
Admin Page MUI Creator Page MUI

### CODE REVIEW

**Comment by taylorpapke** [comment]
Don't know if you were going to do this but looks like you just need to do an npm install of the 2 modules shown in the error message below and then push to this PR branch:
Screenshot 2024-05-05 at 2 35 20 PM

**Comment by MaddasaHatter** [comment]
Just like the other PR it's failing the CI checks and I'm bumping into the above issue as stated by Taylor.

**Comment by MuhammadNSC** [comment]
I've ran npm install several times for this PR, but i keep bumping into dependency errors.

**Review by Asfand00 — COMMENTED** [review]
admin-page

products2
Same Issues as your creator page are happening here as well (for your other PR). 

Also having the same issue with the check not going through. I'm sure once you make the changes on your creator page, you should apply them here as well and both PRs will be good.

**Comment by Aki047** [comment]
I am getting this error when I run the changes made in the branch

image

**Review by BradleyCharles — COMMENTED** [review]
Unfortunately I am running into the same issues
image

**Review by alasali1 — DISMISSED** [review]
I am getting the same error as the others though it is strange because I am seeing the @mui folder in node_modules, though maybe not all the dependencies are getting carried over through github?
image

**Review by nwm516 — COMMENTED** [review]
I changed the import statements to match with MUI 5 since that is what we are using. I also removed a dependency within package.json that still was tied to "@material-ui/core": "^4.12.4", which looks like it was causing the checks not to pass.

**Review by nwm516 — COMMENTED** [review]
I ended up changing a few things within this: 

1. Ended up deleting yarn.lock due to dependency issues running between npm and yarn. Didn't realize that was gonna present itself as a problem but now I know.
2. Changed the import statements for the components to match with MUI 5, since that is what we are now using.
3. Removed the "@material-ui/core": "^4.12.4" from the package.json file, which was causing the initial CI check to fail when running within this pull request.

**Review by taylorpapke — APPROVED** [review]

**Comment by cshimm** [comment]
Looks good on my end! No errors, everything compiles. 
image

**Review by Asfand00 — APPROVED** [review]
LGTM! everything checks out on my end.
admin-page2

File: app/admin-page/page.tsx
Comment (gitbiruk2010) [inline app/admin-page/page.tsx:1]: Code Seems clear and well structured. Good documentations on the screenshots to help visualize the changes.

**Review by intisarosman1 — APPROVED** [review]
Everything looks and
Screenshot 2024-05-16 215133
 works as it should

**Review by heosman — APPROVED** [review]
Looks good to me!

**Review by nickolasram — APPROVED** [review]
Everything works on my end. Great job getting it working 👍

**Pull request merged**